contrib/python/scripts/pyvista_2D_annulus_example.py

```diff
@@ -0,0 +1,260 @@
+# Overview (description from John Nabiloff pyvista_2D_example scripts)
```

Comment:
Please provide an overview of what your script does first, then at some point near the end say refer to /contrib/python/scripts/pyvista_2D_example.py) for more information about PyVista and related installation instructions.

Reply:
In your description of this model, please not that this script uses the time reader!